notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb

```diff
@@ -2,7 +2,7 @@
   "cells": [
```

Comment:
<p>This looks like the wrong cell type</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>